util/topsql/reporter/reporter.go

```diff
@@ -62,6 +65,23 @@ type dataPoints struct {
 	CPUTimeMsTotal uint64
 }
 
+func (d *dataPoints) isInvalid() bool {
```

Comment:
`isInvalid` function only use to check `len(d.TimestampList) != len(d.CPUTimeMsList)`, it doesn't care about the sort.
Normally it will be always valid, but if some other code has bug that make it is invalid, without this check will cause panic.

Reply:
No, I mean "dataPoints". I just cannot put my comment at the place where dataPoints is defined.

Reply:
For example, in the Golang example the sorting behavior is known by the name https://play.golang.org/p/LZgZi-s8IX-